### ISSUE TITLE
Large clipboard data freezes all input

### ISSUE THREAD

**Issue by ehfd** [selkies-project/selkies-gstreamer#43]
```
INFO:webrtc_input:sending clipboard content, length: 49194
0:05:03.202935604   197 0x7f0f90002b80 WARN       webrtcdatachannel webrtcdatachannel.c:264:_channel_store_error:<WebRTCDataChannel@0x1e04db0> Error: Requested to send a string that is too large
0:05:03.203222887   197 0x7f0f980022a0 WARN       webrtcdatachannel datachannel.c:449:gst_webrtc_data_channel_on_error:<WebRTCDataChannel@0x1e04db0> Error: Requested to send a string that is too large
TypeError: <lambda>() takes 1 positional argument but 2 were given
WARNING:gstwebrtc_app:unhandled on_data_close
```

To reproduce: Ctrl+C anything that is sufficiently long. All input becomes impossible until session is reloaded.

**Comment by ehfd** [comment]
Should set a maximum on clipboard length and don't send it if so.
Else, the exception should be handled gracefully.
Relevant code: https://github.com/selkies-project/selkies-gstreamer/blob/v1.3.1/src/selkies_gstreamer/gstwebrtc_app.py#L753

**Comment by ehfd** [comment]
Patched by #73.

HOWEVER, For this issue to be really fixed, we need to implement DataChannel chunking. This is moved over to issue #74.
